mne/beamformer/_lcmv.py

```diff
@@ -27,7 +26,7 @@
 @verbose
 def make_lcmv(info, forward, data_cov, reg=0.05, noise_cov=None, label=None,
               pick_ori=None, rank='info', weight_norm='unit-noise-gain',
-              reduce_rank=False, verbose=None):
+              reduce_rank=False, depth=None, verbose=None):
```

Comment:
I would advocate for _not_ naming the parameter ``depth``. I know this is consistent with minimum norm, however, it is not common terminology in beamforming and will probably be confusing for a lot of users, especially if they are not familiar with minimum norm. ``fwd_norm`` or alike is better. (Even if not public option yet.)

Reply:
I disagree here, to me within-package consistency takes priority. It's easy enough for users to come to understand what `depth` does. Moreover, we might want to add `loose`, which e.g. for `loose=0.8` also does a weighting/normalization of the foward operator; or MRI weighting, which also weights the forward operator. So `fwd_norm` is also less future compatible.

Reply:
(in fact I think in the long run we should deprecate `fwd_norm` in DICS for these reasons)

Reply:
... maybe adding to the docstring that this param is called `fwd_norm` elsewhere (FT?) would help?

Reply:
I've pushed a commit that will hopefully make it clearer in the docstring (for users who are more familiar with the [FT wording](http://www.fieldtriptoolbox.org/tutorial/beamformer/?&#the_forward_model_and_lead_field_matrix)) while keeping our API uniform / moving it toward uniformity:

https://11798-1301584-gh.circle-artifacts.com/0/dev/generated/mne.beamformer.make_lcmv.html#mne.beamformer.make_lcmv